app/frontend/src/components/base/BaseSecure.vue

```diff
@@ -32,15 +32,15 @@ export default {
       'createLoginUrl',
       'isAdmin',
       'keycloakReady'
-    ]),
-    authorized() {
-      return this.admin ? this.isAdmin : true;
-    }
+    ])
   },
   methods: {
+    authorized() {
+      return this.admin ? this.isAdmin : true;
```

Comment:
so does this mean only users in the 'Realm Administrator' group can see the secured ui?

Reply:
This authorized function is actually the same as the previous version where it existed as a computed attribute. This is only checking to see if a user has the GETOK_ADMIN role on the realm - this is different than the Realm Administrator role you're thinking about.

I moved this authorized function to methods because if/when we start having multiple roles we have to check on, it needs to be calculated during runtime and cannot be cached (ref silvipc authorized).

Reply:
ok. thanks. good to know. i'll take another look at that